etc/galasa-tests/com.ibm.cics.cip.cbsa.galasa.tests.install/src/main/java/com/ibm/cics/cip/cbsa/galasa/tests/install/TestAccount.java

```diff
@@ -0,0 +1,581 @@
+			assertThat(terminal.retrieveScreen()).contains("Amount successfully applied to the account");
+
+			String postCreditValue = loanAccount.getAccountValue();
+			assertThat(!(initialValue.equals(postCreditValue)));
+		} catch (Exception e) {
+			throw new CbsaException("Failed to credit funds to an account", e);
+		}
+	}
+
+
+	/*
+	 * This test checks that funds can be debited to an account successfully
+	 */
+	@Test
+	public void debitFunds() throws Exception {
+		try {
+			String initialValue = loanAccount.getAccountValue();
+
+			terminal.goToScreen(CbsaMenuItem.CREDIT_FUNDS);
+			terminal.waitForKeyboard().positionCursorToFieldContaining("ACCOUNT NUMBER").tab().type(loanAccount.getAccountNumber()).waitForKeyboard().tab().type("-").waitForKeyboard().type("0000005000.00").enter().waitForKeyboard();
+			assertThat(terminal.retrieveScreen()).contains("Amount successfully applied to the account");
+
+			String postDebitValue = loanAccount.getAccountValue();
+			assertThat(!(initialValue.equals(postDebitValue)));
+		} catch (Exception e) {
+			throw new CbsaException("Failed to debit funds to an account", e);
+		}
+	}
+
+
+	/*
+	 * This test checks that funds cannot be credited to a non-existent account
+	 */
+	@Test
+	public void creditNonExistentAccount() throws Exception {
+		try {
+			String initialValue = loanAccount.getAccountValue();
+
+			// Takes string, turns into long, adds 500, turns back to string
+			String invalidAccountNumber = Long.toString(Long.parseLong(loanAccount.getAccountNumber())+500);
+
+			terminal.goToScreen(CbsaMenuItem.CREDIT_FUNDS);
+			terminal.waitForKeyboard().positionCursorToFieldContaining("ACCOUNT NUMBER").tab().type(invalidAccountNumber).waitForKeyboard().tab().type("+").waitForKeyboard().type("0000005000.00").enter().waitForKeyboard();
+			assertThat(terminal.retrieveScreen()).contains("Amount not applied");
+
+			String postCreditValue = loanAccount.getAccountValue();
+			assertThat(initialValue.equals(postCreditValue));
+		} catch (Exception e) {
+			throw new CbsaException("Failed to credit funds to a non-existent account", e);
+		}
+	}
+
+
+	/*
+	 * This test checks that funds cannot be debited to a non existent account
+	 */
+	@Test
+	public void debitNonExistentAccount() throws Exception {
+		try {
+			String initialValue = loanAccount.getAccountValue();
+
+			// Takes string, turns into long, adds 500, turns back to string
+			String invalidAccountNumber = Long.toString(Long.parseLong(loanAccount.getAccountNumber())+500);
+
+			terminal.goToScreen(CbsaMenuItem.CREDIT_FUNDS);
+			terminal.waitForKeyboard().positionCursorToFieldContaining("ACCOUNT NUMBER").tab().type(invalidAccountNumber).waitForKeyboard().tab().type("-").waitForKeyboard().type("0000005000.00").enter().waitForKeyboard();
+			assertThat(terminal.retrieveScreen()).contains("Amount not applied");
+
+			String postCreditValue = loanAccount.getAccountValue();
+			assertThat(initialValue.equals(postCreditValue));
```

Comment:
Here as well - use `assertThat(...).isEqualTo(...)`


Same applies throughout this file.